File: LUA/Cap06/c06ex04.lua
-- c06ex04.lua

aluno = {}

io.write("Entre a quantidade de elementos da matriz: ")
n = tonumber(io.read())
io.write("\n")

for i = 1, n do
  io.write(string.format("Entre o %3do. nome: ", i))
  aluno[i] = io.read()
end

io.write("\nForam fornecidos os nomes:\n\n")

for i = 1, n do
  io.write(string.format("Nome %3d = %s\n", i, aluno[i]))
end

io.write("\nTecle <Enter> para encerrar... ")
io.read()

File: PYTHON/Cap06/c06ex04.py
# c06ex04.py

aluno = {}

print("Entre a quantidade de elementos da matriz: ", end="")
n = int(input())
print()

for i in range(1, n+1):
  print(f"Entre o {i:3}o. nome: ", end="")
  aluno[i] = input()

print("\nForam fornecidos os nomes:\n")

for i in range(1, n+1):
  print(f"Nome {i:3} = {aluno[i]}")

input("\nTecle <Enter> para encerrar... ")
